src/components/Container/Container.styles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import backgroud from '../../assets/bg.jpg';
+import mobileBackground from '../../assets/bg-mobile.jpg';
+import { breackpoints } from '../../theme';
```

Comment:
typo

Reply:
breackpoints -> breakpoints

Reply:
renamed, update added to the branch "task 6"